sig-network/charter.md

```diff
@@ -0,0 +1,42 @@
+# SIG Network
+
+## Scope
+
+SIG Network is owning everything related to KubevVirt networking.
+The primary objective of the sig is to connect the Virtual Machine to the pod network/s,
+allowing migration and other advantages of virtualization in addition to maintaining services,
```

Comment:
maybe add the word connectivity please ?
allowing "connectivity," migration....

Reply:
Connectivity is not a virtualization advantage. I tried to say we maintain k8s networking features alongside with virtualization ones.

Reply:
@ormergi Can I resolve this comment or is it a blocker from your side?

Reply:
sgtm, you can close, thanks